src14/owned_proxy/README.md

```diff
@@ -0,0 +1,57 @@
+# SRC-14: Owned Proxy Contract
+
+- The [Owned Proxy Contract](./contract/src/main.sw) is an opinionated implementation of the [extended SRC-14 standard](https://docs.fuel.network/docs/sway-standards/src-14-simple-upgradeable-proxies/). It utilises the [Upgradability library from sway-libs](https://github.com/FuelLabs/sway-libs) and includes initialization functionality that allows for secure ownership upon deployment.
+
```

Comment:
Lets add a warning here saying that this utils scripts are temporary and will be integrated into forc in the near future.